From 9312965e31de62c81b21ea166e6fa03161b3e5ca Mon Sep 17 00:00:00 2001
From: Ioana Radulescu <ruxandra.radulescu@nxp.com>
Date: Mon, 9 Jul 2018 19:14:16 +0300
Subject: [PATCH 411/448] staging: fsl-dpaa2/eth: Enable debugfs support by default

The debugfs counters of the DPAA2 Ethernet driver are useful in
many occasions and bring no runtime overhead, so it makes sense
to enable them by default.

Signed-off-by: Ioana Radulescu <ruxandra.radulescu@nxp.com>
---
 drivers/staging/fsl-dpaa2/Kconfig |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/drivers/staging/fsl-dpaa2/Kconfig b/drivers/staging/fsl-dpaa2/Kconfig
index 6b6dd86..cab0d03 100644
--- a/drivers/staging/fsl-dpaa2/Kconfig
+++ b/drivers/staging/fsl-dpaa2/Kconfig
@@ -31,7 +31,7 @@ config FSL_DPAA2_ETH_USE_ERR_QUEUE
 config FSL_DPAA2_ETH_DEBUGFS
 	depends on DEBUG_FS
 	bool "Enable debugfs support"
-	default n
+	default y
 	---help---
 	  Enable advanced statistics through debugfs interface.
 
-- 
1.7.1

